### DISCUSSION TITLE
Kratos Conflicting Dependencies

### DISCUSSION THREAD

**Discussion by OMERTBITU** [KratosMultiphysics/Kratos#9978] [Q&A] [answered]
Hello all.
I am new about Gid and Kratos. I am trying to setup newest developer version of Kratos via Github documents and the Youtube video about developer setup. In the last part, program says "Python 3.10.5 installed, but there are some missing packages. Do you want to Kratos to install them? Packages to be installed KratosMultiphysics-all". So I choose "yes". And it gives error messages as there are conflicting dependencies. KratosMultiphysics-all9.1.3,9.1.2,9.1.1,etc. What do i need to do?
Thanks.

**Comment by rubenzorrilla** [comment]
@jginternational

**Comment by jginternational** [comment, marked as answer]
Dear @OMERTBITU 

Open a console and run 

`pip install KratosMultiphysics-all --upgrade`

This will upgrade the Kratos pip packages to the latest version.

Anyway, I'm planning to deploy this week a new GiD Kratos release with some fixes

**Reply by OMERTBITU** [reply]
Thank you.
